README.md

```diff
@@ -2,5 +2,28 @@
 
 [NOTE: This is not an officially supported Google product.]
 
-Raksha is a project to build a unified framework for specifying and
+Raksha[^1] is a project to build a unified framework for specifying and
 enforcing privacy policies in a system with heterogeneous runtimes.
+
+## Building
+
+On a Linux Debian system, for example:
+
+% sudo apt-get install bazel mcpp
+% bazel build ...
```

Comment:
These are rendering on the same line. You might want to add a new line between them?

Reply:
Nice catch, I fixed by wrapping with a preformatted-code backtick block.